t/18_saputils.t

```diff
@@ -25,9 +25,9 @@ Global/global/maintenance="false"
 Sites/site_b/b="SOK"
 Hosts/vmhana01/vhost="AAAAAAA"
 Hosts/vmhana02/vhost="BBBBBBB"');
-    ok keys %$topology == 2, 'Output is about exactly 2 hosts';
-    ok((any { qr/vmhana01/ } keys %$topology), 'External hash has key vmhana01');
-    ok((any { qr/vmhana02/ } keys %$topology), 'External hash has key vmhana02');
+    ok keys %{$topology->{'Host'} == 2, 'Output is about exactly 2 hosts';
+    ok((any { qr/vmhana01/ } keys %{$topology->{'Host'}}), 'External hash has key vmhana01');
+    ok((any { qr/vmhana02/ } keys %{$topology->{'Host'}}), 'External hash has key vmhana02');
 };
 
 subtest '[calculate_hana_topology] internal keys' => sub {
```

Comment:
Please add a new test about `calculate_hana_topology` to use integer value for `node_state`